### ISSUE TITLE
The PCK results (run the mainRigidPck.m) are much better than the result in the original paper

### ISSUE THREAD

**Issue by qingnanli** [shubhtuls/ViewpointsAndKeypoints#4]
Hi, shubhtuls, I met a problem again. 
I run the code provided in your git, and get the following results, which are a little better than the performance in the original paper.
For aeroplane (evaluated on PASCAL 3D, total 865 images),  I have 0.5186, 0.6625, 0.6912, 0.7149 corresponding to conv6, conv12, conv6+conv12 and conv6+conv12+pLikelihooh  vs 51.4, 54.9, 61.9, 66.0 in the original paper.
For bicycle (evaluated on PASCAL 3D, total 711 images), I have 0.6303, 0.7431, 0.7759, 0.7905 whereas the results in original paper are 62.4, 66.8, 74.6, 77.8.
For boat (evaluated on PASCAL 3D, total 850 images), I have 0.4436, 0.5036, 0.5506, 0.5830 vs 37.8, 32.6, 43.6, 52.1 in the original paper.
I keep the params.alpha = 0.1, and is there something wrong?

**Comment by shubhtuls** [comment]
Hi,
The code was released a couple months after the camera ready so due to some cleaning up and minor changes in training, I remember the results improving a bit (but not so significantly). Could you report the mean performance across classes for 'conv6+conv12+pLikelihood' that you obtain ?

Followup comment : I'm not sure if you modified the evaluation script in some way but the number of instances/images you mention seem very high e.g. 865 aeroplane instances/images - do these also include the training set?

**Comment by qingnanli** [comment]
Thanks very much! I made a mistake :( 
The 865 aeroplane images include the validation set as well as the training set.

**Comment by qingnanli** [comment]
I modified the number of testing instances according to the areoplane.mat in the cahedir/evalSets at the start time. Therefore, we had 389 testing instances (12 occluded instances, a total of 401 testing instances ). But the results were not changed. Because we found that the last few lines in mainRigidPck.m had already abandoned the training data and occluded data. 
The mean performance across classes for 'conv6+conv12+pLikelihood' is 71.97.

**Comment by shubhtuls** [comment]
Ok, thanks for reporting the number. The number 71.9% is only about a 3% improvement in the results mentioned in the paper which seems possible given some small changes in training during code release. I don't think you're doing anything wrong in that scenario. If you're interested in comparing to/reporting the number, I'd recommend (if possible) retraining the network(s) again with a different seed and using the mean.

**Comment by qingnanli** [comment]
Thanks very very much for your help! Closing the issue now :)
